cookbook/deployment/heroku.rst

```diff
@@ -305,20 +305,20 @@ This is also very useful to build assets on the production system, e.g. with Ass
 
     With the next deploy, Heroku compiles your app using the Node.js buildpack and
     your npm packages become installed. On the other hand, your ``composer.json`` is
-    now ignored. To compile your app with both buildpacks, Node.js *and* PHP, you can
-    use a special `multiple buildpack`_. To override buildpack auto-detection, you
-    need to explicitly set the buildpack URL:
+    now ignored. To compile your app with both buildpacks, Node.js *and* PHP, you need
+    to use both buildpacks. To override buildpack auto-detection, you
```

Comment:
I would rather use something like "To compile your app with both buildpacks, manually select them with the `buildpacks:set` command" (because the key concept here isn't that you're overriding the auto-detection, but rather that you're using multiple packs).